client/app/scripts/liveblog-edit/module.js

```diff
@@ -239,9 +242,25 @@ define([
         api('users').getById(blog.original_creator).then(function(data) {
             vm.original_creator = data;
         });
+        api('users').query().then(function(data) {
+            vm.avUsers = data._items;
```

Comment:
I'm not sure this is the right way to retrieve the users. Think if we have more than one page of result.
I think we should use a field with auto completion making directly requests to the server